docs/digger/admin.asciidoc

```diff
@@ -5,12 +5,12 @@ title: Digger Administration guide
 toc_generate: true
 ---
 
-include::docs/digger/admin/monitoring.asciidoc[]
+include::/admin/digger-nagios-monitoring.asciidoc[]
 
-include::docs/digger/admin/use-http-proxy.asciidoc[]
+include::/admin/using-http-proxy.asciidoc[]
 
-include::docs/digger/admin/manage-artifacts.asciidoc[]
+include::/admin/managing-digger-logs-artifacts.asciidoc[]
 
-include::docs/digger/admin/installing-plugins.asciidoc[]
+include::/admin/installing-jenkins-plugins.asciidoc[]
 
-include::docs/digger/admin/known-issues.asciidoc[]
+include::/admin/known-issues.asciidoc[]
```

Comment:
@wei-lee it is not needed for lang. style preview, atom is sufficient, will push correct links after the work is finished. 

Just a headsup: teh guide is not 100% correct or it just did not work on my machine.